godtools/App/Features/ToolShortcutLinks/Data-DomainInterface/GetToolShortcutLinksRepository.swift

```diff
@@ -25,7 +25,7 @@ class GetToolShortcutLinksRepository: GetToolShortcutLinksRepositoryInterface {
     
     func getLinksPublisher(appLanguage: AppLanguageDomainModel) -> AnyPublisher<[ToolShortcutLinkDomainModel], Never> {
         
-        return favoritedResourcesRepository.getFavoritedResourcesSortedByCreatedAtPublisher(ascendingOrder: false)
+        return favoritedResourcesRepository.getFavoritedResourcesSortedByPositionPublisher(ascendingOrder: false)
```

Comment:
For tool shortcuts make sure it matches the favorited list order.  Ascending defaults to true so you will want to remove this flag.